recipe_scrapers/_utils.py

```diff
@@ -274,3 +281,14 @@ def change_keys(obj, convert):
 def get_url_slug(url):
     path = url_path_to_dict(url).get("path")
     return path.split("/")[-1]
+
+
+def unsupported_field_warning(host, field):
```

Comment:
Recommend following the same approach as #1077 for this; only implement for the affected scraper, not as a _utils` module feature yet.

It _probably_ makes sense to share a common code path for these kind of warnings at some point, but let's begin simple.